test/scan.test.ts

```diff
@@ -107,7 +107,7 @@ test('should allow for filterExpression to be provided', async () => {
 async function runSegmentTillFinished(
   context: TestContext,
   segment: number,
-  totalSegments: number,
+  totalSegments: number
```

Comment:
nit: it looks like a prettier misconfig on this project - we should be using trailing commas everywhere

Reply:
:o nice catch, I just ran `yarn format` and those were the changes. This is something I can probably take a look in another PR.